### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kathsherratt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,65 +12,92 @@
...
-See the individual function documentation for full details of present functionality. The main function imports and cleans the linelist. 
+### Worldwide data
 
-```r
-NCoVUtils::get_linelist()
-```
+There are three sources of worldwide, country-level data on cases. One also includes deaths.
 
-Extract international case counts compiled by WHO using:
+1. Extract total global cases and deaths by country using:
```
Comment (ffinger) [inline README.md]: what is the source of the data for the `get_total_cases` function?

File: README.md [hunk 2]
```diff
@@ -12,65 +12,92 @@
...
+  + ```NCoVUtils::get_total_cases()```
+2. Extract international case counts compiled by the WHO using:
```
Comment (ffinger) [inline README.md]:
```suggestion
2. Extract daily international case and death counts compiled by the WHO using:
```

File: README.md [hunk 3]
```diff
@@ -12,65 +12,92 @@
...
+  + ```NCoVUtils::get_who_cases()```
+3. Extract international case counts compiled by ECDC using:
```
Comment (ffinger) [inline README.md]:
```suggestion
3. Extract daily international case and death counts compiled by ECDC using:
```

File: README.md [hunk 4]
```diff
@@ -12,65 +12,92 @@
...
-Extract total global cases (and deaths) by country using:
+* ```NCoVUtils::get_linelist()```
 
-```r
-NCoVUtils::get_total_cases()
-```
+### Sub-national data
 
-Extract non-pharmaceutical interventions by country using:
+We currently have functions to extract sub-national level data for 11 countries. These are typically at the admin-1 level, the largest regions available. We are also working on joining the data to standard georeferencing codes to allow easy mapping.
```
Comment (ffinger) [inline README.md]: I wouldn't put the number of countries, we will forget to update it.

Comment (ffinger) [inline README.md]:
```suggestion
We currently have functions to extract sub-national level data for 11 countries. These are at the most detailed subdivision level available, typically adm1 (provinces, regions, states or similar). We are also working on adding standard georeferencing codes to allow easy mapping.
```

File: README.md [hunk 5]
```diff
@@ -12,65 +12,92 @@
...
 
-```r
-NCoVUtils::get_interventions_data()
-```
+For most countries, subnational cases can be extracted by using the following:
```
Comment (ffinger) [inline README.md]: I wouldn't say this, there is too many exceptions...
Maybe we should work on a standardized interface and/or a wrapper function that takes "country" as an argument and calls the respective function.

File: README.md [hunk 6]
```diff
@@ -12,65 +12,92 @@
...
-* France  
-* Germany  
-* Italy  
-* Spain  
-* United Kingdom  
-* United States  
-* Japan  
-* Korea
-* Afghanistan
+We are working to include more countries.
```
Comment (ffinger) [inline README.md]:
```suggestion
We are working to include more countries.
Contributions via Pull Request are most welcome.
```

**Review by ffinger — CHANGES_REQUESTED** [review]
Looks good to me, some comments on the wording.

File: README.md (hunk 5 above)
Comment (ffinger) [reply README.md]: > Maybe we should work on a standardized interface and/or a wrapper function that takes "country" as an argument and calls the respective function.

Added an issue for this.

**Review by seabbs — APPROVED** [review]
Looks good

**Review by seabbs — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -12,65 +12,86 @@
...
 
 ## Usage
 
-See the individual function documentation for full details of present functionality. The main function imports and cleans the linelist. 
+### Worldwide data
 
-```r
-NCoVUtils::get_linelist()
-```
+There are three sources of worldwide, country-level data on cases and deaths.
```
Comment (ffinger) [inline README.md]: I see only two? There is three functions but two sources?

File: README.md [hunk 8]
```diff
@@ -12,65 +12,86 @@
...
-* United Kingdom  
-* United States  
-* Japan  
-* Korea
-* Afghanistan
+We are working to include more countries.
 
-See the function reference for a full list of data sets that package can be used to extract.
 
 ## Development
```
Comment (ffinger) [inline README.md:90]: maybe add sentence that help is appreciated, see issues? or describe some domains where help is appreciated (new data sources, geo-coding, ....)?

**Review by ffinger — COMMENTED** [review]
Very minor requests, feel free to ignore.

**Review by ffinger — APPROVED** [review]

**Comment by kathsherratt** [comment]
Thanks very much for reviewing twice Flavio (I'm sorry for taking a long time earlier to update this). I've included the new comments so this should be ready to go. Thanks again!

**Comment by ffinger** [comment]
There seems to be a build failure in the tests, but nothing to do in with the readme.

**Pull request merged**